frontend/.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.tabSize": 4,
```

Comment:
Don't think i agree with this change. Usually all across our project we follow `tabSize: 2`.

Reply:
💯  agree... let's not make this change.

Reply:
Removed the line, will included it in lower tree

Reply:
> Removed the line, will included it in lower tree

We should have uniform tab size settings for the same file types across the repo. What files have a 4 space tab? For JavaScript and TypeScript, we should use prettier.

Reply:
This is for Gherkin files `.feature` -- BDD test files that QE creates @spadgett -- Which is why we wanted to avoid the change here.

Reply:
@spadgett - I think approval is required from you, As this pr contains modifications related to console/.git-ignore file